L1Trigger/L1CaloTrigger/interface/Phase2L1RCT.h

```diff
@@ -1245,7 +1245,7 @@ inline p2eg::clusterInfo p2eg::getBremsValuesNeg(p2eg::crystal tempX[p2eg::CRYST
 
   // Initialize all entries in a new ((15+2)x(20+4)) array to be zero.
```

Comment:
Just a very small comment—would it be possible to update the note here to match the code and avoid confusion? It should read ((15+2) x (20+7)), correct?"

Reply:
Thanks for the catch @Moanwar, I updated the commit to also change the comment.